DependencyInjection/Configuration.php

```diff
@@ -350,10 +350,86 @@ private function addOrmSection(ArrayNodeDefinition $node)
                     ->append($this->getOrmEntityManagersNode())
                     ->fixXmlConfig('resolve_target_entity', 'resolve_target_entities')
                     ->append($this->getOrmTargetEntityResolverNode())
+                    ->append($this->getOrmSecondLevelCacheNode())
                 ->end()
             ->end()
         ;
     }
+    
+    /**
+     * Return ORM second level node
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\NodeDefinition
+     */
+    private function getOrmSecondLevelCacheNode()
+    {
+        $treeBuilder = new TreeBuilder();
+        $node = $treeBuilder->root('second_level_cache');
+
+        $node
+            ->info('Global second level cache defaults (for all entity managers).')
```

Comment:
IMO, we should not have a global config for them. The config should be separate for each entity manager.

Btw, if you want to keep it global, you are missing the code whitelisting the key in the logic handling the shortcut config